lms/djangoapps/grades/tests/test_course_grade_factory.py

```diff
@@ -286,7 +286,7 @@ def test_grading_exception(self, mock_course_grade):
             else mock_course_grade.return_value
             for student in self.students
         ]
-        with self.assertNumQueries(8):
+        with self.assertNumQueries(11):
```

Comment:
Same comment here about the query increases.

Reply:
idem. Out of scope in this PR